src/Select.js

```diff
@@ -866,6 +866,25 @@ class Select extends React.Component {
 		);
 	}
 
+  renderBottomClear() {
+    if (!this.props.clearable || this.props.value === undefined || this.props.value === null || this.props.multi && !this.props.value.length || this.props.disabled || this.props.isLoading) return;
+    const clear = this.props.clearAllText;
+    return (
+      <div className="Select-multi-clear-wrapper">
+        <span className="Select-multi-clear-zone"
+          title={this.props.clearAllText}
+          aria-label={this.props.clearAllText}
+          onMouseDown={this.clearValue}
```

Comment:
Shouldn't the value be cleared with `onMouseUp`? It seems a bit odd to clear the values as soon you click the button and before you release it.